editor/node_3d_editor_gizmos.cpp

```diff
@@ -2350,23 +2792,22 @@ String GPUParticles3DGizmoPlugin::get_handle_name(const EditorNode3DGizmo *p_giz
 		case 5:
 			return "Pos Z";
 	}
-
 	return "";
 }
 
-Variant GPUParticles3DGizmoPlugin::get_handle_value(EditorNode3DGizmo *p_gizmo, int p_idx) const {
+Variant GPUParticles3DGizmoPlugin::get_handle_value(const EditorNode3DGizmo *p_gizmo, int p_group_id, int p_id) const {
 	GPUParticles3D *particles = Object::cast_to<GPUParticles3D>(p_gizmo->get_spatial_node());
 	return particles->get_visibility_aabb();
 }
 
-void GPUParticles3DGizmoPlugin::set_handle(EditorNode3DGizmo *p_gizmo, int p_idx, Camera3D *p_camera, const Point2 &p_point) {
+void GPUParticles3DGizmoPlugin::set_handle(const EditorNode3DGizmo *p_gizmo, int p_group_id, int p_id, Camera3D *p_camera, const Point2 &p_point) const {
 	GPUParticles3D *particles = Object::cast_to<GPUParticles3D>(p_gizmo->get_spatial_node());
 
 	Transform3D gt = particles->get_global_transform();
 	Transform3D gi = gt.affine_inverse();
 
-	bool move = p_idx >= 3;
-	p_idx = p_idx % 3;
+	bool move = p_id >= 3;
+	p_id = p_id % 3;
```

Comment:
Here can be >= 3, so we can't use Vector3::Axis to store all values, but I'm wondering if instead that we should make moving or not be a boolean or enum that indicates the type of action being taken instead of using the values 3 to 5.

Reply:
The "group ID" feature added in this PR would help with that, since you can have a separate group for moving and resizing, but after some discussion with Juan it's going to be removed in order to keep things simpler to use.